### ISSUE TITLE
Wrong import in web.howto.web-in-60.error-handling.html

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6503]
|<img alt="introom's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| introom reported|
|-|-|
|Trac ID|trac#6503|
|Type|defect|
|Created|2013-05-03 04:16:20Z|

It writes: from twisted.web.error import NoResource
But I think it should be: from twisted.web.resource import NoResource

Attachments:

* [6503.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/65a/65a58e108b8bee6efc51ef9b2f8deb21f6be4361/18efc1f97e1d866d4840d5ad077631177ec1076e.patch) (1101 bytes) - added by introom on 2013-05-03 04:23:41Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6503 6503
type__defect defect
reporter__introom introom
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation documentation
time__1367554580000000 1367554580000000
changetime__1368410511000000 1368410511000000
version__None None
owner__thijs thijs
cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks for your patch, I'll apply it.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [38401]) Apply 6503.patch: Correct the NoResource import.

Author: introom
Reviewer: thijs
Fixes: [#6503](https://github.com/twisted/twisted/issues/6503)

The deprecated alias in t.web.error was removed a while ago.
